N24DCCN058/Bai tap buoi 8/Bai6.cpp

```diff
@@ -0,0 +1,20 @@
+#include <bits/stdc++.h>
+
+using namespace std ;
+
+string CountStr(string str){
+    string count ;
+    for (int i = 0 ; i < str.size() ; i++){
+        if ( str[i] != ' '){
+            count += str[i] ;
+        }    
+    }
+    return count ;
+}
+ 
+int main() {
+    string n ;
+    getline(cin , n) ;
+    cout << CountStr(n) ;
```

Comment:
In ra đúng định dạng của output nha [“Cau”,“lac”, “bo”, “ITMC”]. Đừng nghĩ nó không quan trọng. Vì thi là chấm máy, nên output phải giống 100% output của đề mới chấm là đúng. nên cẩn thận đừng chủ quan nhá